Resources/Audio/_White/Lobby/attributions.yml

```
@@ -0,0 +1,79 @@
+  copyright: "Space Oddity by Chris Hadfield"
+  source: "https://youtu.be/KaOC9danxNo"
+
+- files: ["space-shuttle.ogg"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Space Shuttle by Эдипов Комплекс"
+  source: "https://youtu.be/aOFjbkAHEGg"
+
+- files: ["title3.ogg"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Tintin on the Moon by Jeroen Tel"
+  source: "youtu.be/Hz6KO0gnrS8"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)